python/tests/integration/connectors/memory/test_milvus.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import numpy as np
+import pytest
+
+from semantic_kernel.connectors.memory.milvus import MilvusMemoryStore
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+try:
+    from milvus import default_server  # noqa: F401
+
+    default_server.start()
+    URI = "http://127.0.0.1:" + str(default_server.listen_port)
+    TOKEN = None
+    milvus_installed = True
```

Comment:
If the server is already running on the machine, milvus_installed will resolve as False. So running the tests twice on the same machine will be successful on the first run, but will skip on the second. I'd recommend some test teardown logic. Test-scoped is easier, but if the server setup takes too long, module scoped could work as well.

Reply:
Good point,resolved